hack/timestamps.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
+
+function timestamps::time_wrapper() {
+    while IFS= read -r line; do
+        printf "%(%T)T: %s\n" "-1" "$line"
+    done
+}
+
+if [[ "${BASH_SOURCE[0]}" == "$0" ]]; then
+    /bin/sh "$@" > >(timestamps::time_wrapper) 2> >(timestamps::time_wrapper)
```

Comment:
stderr redirection should be improved to keep stdout and stderr separated:

`2> >(timestamps::time_wrapper >&2)`

Reply:
Nice catch! fixed, thanks.